### ISSUE TITLE
Jury Conditions vs. Eigenvalues < 1

### ISSUE THREAD

**Issue by llaskowski96** [bbolker/math3mb#9]
A fundamental question that I would just like confirmed: Applying the Jury Conditions is equivalent to seeing if the eigenvalues of a matrix M are less than one to confirm stability for a given equilibrium.

**Comment by bbolker** [comment]
Yes. Jury conditions in the simple form given (|T| < 1+ Delta < 2, or equivalently |T| < 1 + Delta *and* Delta < 1) only work for 2-dimensional systems/2x2 matrices, though. Otherwise you have to do something [more complicated](https://en.wikipedia.org/wiki/Jury_stability_criterion).
